ios/Openid4vpBle/Openid4vpBle.swift

```diff
@@ -38,12 +38,20 @@ class Openid4vpBle: RCTEventEmitter {
         }
         return dictonary!
     }
-    
+
     @objc
     func getConnectionParametersDebug() -> String {
         return "GetConnectionParametersDebug"
     }
 
+    @objc(setTuvaliVersion:)
```

Comment:
Since the function should be private to Tuvali layer, do we need this annotation?

Reply:
we are calling setTuvaliVersion() from index.tsx and to expose kotlin methods to js layer we have to annotate it with "@ReactMethod " annotation and if we don't add annotation we are not able to access it in index.tsx. But we tried searching for ways how to use this function only within our project and exclude it when adding tuvali as dependency in inji and tried different annotation @hide and RestrictTo and also made method as private but still we are able to access it in inji